modules/ossm-cr-example.adoc

```diff
@@ -18,63 +18,257 @@ This example `ServiceMeshControlPlane` definition contains all of the supported
+      deployment:
+        replicas: 2
+      pod:
+        affinity: {}
+      container:
+        resources:
+          limits: {}
+          requirements: {}
 
-    kiali:
-      enabled: true
+  addons:
+    visualization:
+      grafana:
+        address: some-grafana-url # use existing grafana install, or...
+        install: # install grafana if not null
+          selfManaged: true # use grafana CR, maybe poor naming?
+          config:
+            env: {} # .Values.grafana.env
+            envSecrets: {} # .Values.grafana.envSecrets
+          persistence:
+            storageClassName: ""
+            accessMode: ReadWriteOnce
+            capacity: 5Gi # not supported in charts
+          service: # similar to addons.tracing.jaeger.install.config.service
+            ingress:
+              contextPath: /grafana
+              tls:
+                termination: reencrypt
+          runtime:
+            deployment: {}
+            pod: {}
+      kiali:
+        name: kiali # kiali CR to create/reference
+        install: # install kiali CR if not present
+          config:
+            dashboard:
+              viewOnly: false
+              enableGrafana: true
+              enableTracing: true
+              enablePrometheus: true
+          service: 
+            ingress:
+              contextPath: /kiali
+          runtime:
+            deployment: {}
+            pod: {}
+----
 
-    grafana:
-      enabled: true
+The following table lists the parameters for the `ServiceMeshControlPlane` resource.
 
-    tracing:
-      enabled: true
-      jaeger:
-        template: all-in-one
-----
+.`ServiceMeshControlPlane` resource parameters
+|===
+|Name |Description |Type
+
+|`apiVersion`
+|APIVersion defines the versioned schema of this representation of an object. Servers convert recognized schemas to the latest internal value, and may reject unrecognized values. The value for {ProductName} version 2.0 is `maistra.io/v2`.
+|string
+
+|kind
+|Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated.
+|CamelCase string
+
+|metadata
+|Type of installation. You can provide a name for your deployment to keep track of your work for example, `basic-install`, or `production`.
+|string
+
+|spec
+|The specification of the desired state of this `ServiceMeshControlPlane`. This includes the configuration options for all components that comprise the control plane.
+|For more information, see `ControlPlaneSpec`.
+
+|status
+|The current status of this `ServiceMeshControlPlane` and the components that comprise the control plane. 
+|For more information, see `ControlPlaneStatus`.
+|===
+
+The following table lists the specifications for the `ServiceMeshControlPlane` resource. These parameters configure {ProductName} for your microservices and apps.
+
+.`ServiceMeshControlPlane` resource spec
+|===
+|Name |Description |Configurable parameters
```

Comment:
The "Configurable parameters" column sometimes indicates the default value instead of the names of subfields. Find a better label for the column?